htdocs/web_portal/views/user/view_user.php

```diff
@@ -122,10 +122,12 @@
                     <tr class="site_table_row_2">
                         <td class="site_table">Home Site</td>
                         <td class="site_table">
-                            <a href="index.php?Page_Type=Site&amp;id=
-                                <?php echo $params['user']->getHomeSite()->getId()?>">
-                                <?php xecho($params['user']->getHomeSite()->getShortName()) ?>
```

Comment:
What was `xecho` doing such that we can now use a plain `echo`?

Reply:
ah good spot, I should use `xecho` as the short name is user submitted input. Fixed in e074ed612e089867a38ad22cc0923d17e8740864.